hs_core/views/__init__.py

```diff
@@ -1,39 +1,45 @@
 from __future__ import absolute_import
 from collections import defaultdict
-import json
-import requests
-
+from django.contrib.auth import login, authenticate
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth.models import Group, User
 from django.contrib.sites.models import Site
 from django.core.exceptions import ValidationError
 from django.http import HttpResponseRedirect, HttpResponse
 from django.shortcuts import get_object_or_404, render_to_response, render
-from django.core import exceptions as ex
 from django.template import RequestContext
-from django.core import signing
-from django import forms
-
+from django.utils.timezone import now
+import json
 from mezzanine.conf import settings
-from mezzanine.pages.page_processors import processor_for
-import autocomplete_light
-from inplaceeditform.commons import get_dict_from_obj, apply_filters
-from inplaceeditform.views import _get_http_response, _get_adaptor
-
+from django import forms
+from mezzanine.generic.models import Keyword
+import mimetypes
+import os
 from hs_core import hydroshare
 from hs_core.hydroshare import get_resource_list
 from hs_core.hydroshare.utils import get_resource_by_shortkey, resource_modified, user_from_id
 from .utils import authorize
 from hs_core.models import ResourceFile, GenericResource, resource_processor, CoreMetaData
+import requests
+from django.core import exceptions as ex
+from mezzanine.pages.page_processors import processor_for
+from django.template import RequestContext
+from django.core import signing
+from django.template import Context
+import django.dispatch
+from django.contrib.contenttypes.models import ContentType
+from django.forms import ValidationError
+from inplaceeditform.commons import get_dict_from_obj, apply_filters, get_adaptor_class
+from inplaceeditform.views import _get_http_response, _get_adaptor
 
-from . import resource_rest_api
-from . import user_rest_api
-
+from . import users_api
+from . import discovery_api
+from . import resource_api
+from . import social_api
```

Comment:
I addressed this by checking out __init__:

```
git checkout 73643b70e3355626ba4993e07649815dd2746c5c -- hs_core/views/__init__.py
```

Why isn't this updated when I pull develop?